sample/src/main/java/com/vimeo/sample/model/Video.java

```diff
@@ -25,13 +25,26 @@
 
 import com.vimeo.stag.GsonAdapterKey;
 
+import java.util.ArrayList;
 import java.util.Date;
 
 public class Video {
 
     @GsonAdapterKey("user")
     public User mUser;
 
+    @GsonAdapterKey("secondUser")
+    public User mSecondUser;
+
+    @GsonAdapterKey
+    ArrayList<User> mUsersList;
+
+    @GsonAdapterKey
+    ArrayList<User> mSecondUsersList;
+
+    @GsonAdapterKey
+    public ArrayList<Stats> mStatsArrayList;
+
```

Comment:
Minor: could we move this to a separate class since these added fields aren't actually part of the Video object returned from the server? Maybe just call it `IdenticalFieldTypes` or something